bootstrap/src/main/java/org/restcomm/media/bootstrap/ioc/provider/mgcp/MgcpChannelInitializerProvider.java

```diff
@@ -43,18 +45,31 @@
     private final MgcpMessageDecoder decoder;
     private final MgcpMessageEncoder encoder;
     private final MgcpChannelInboundHandler inboundHandler;
+    private final MgcpControllerConfiguration controller;
+    private final int mgcpBufferSize;
 
     @Inject
     public MgcpChannelInitializerProvider(@Named("mgcpNetworkGuard") NetworkGuard networkGuard, MgcpMessageParser parser, MgcpChannelInboundHandler inboundHandler) {
         this.filter = new NetworkFilter(networkGuard);
         this.decoder = new MgcpMessageDecoder(parser);
         this.encoder = new MgcpMessageEncoder();
         this.inboundHandler = inboundHandler;
+        this.controller = null;
+        this.mgcpBufferSize = 5000;
+    }
+
+    public MgcpChannelInitializerProvider(MediaServerConfiguration config, @Named("mgcpNetworkGuard") NetworkGuard networkGuard, MgcpMessageParser parser, MgcpChannelInboundHandler inboundHandler) {
```

Comment:
Please remove this constructor. This is a Guice Provider which relies on dependency injection, so only one constructor is accepted.

On a side note, @gsaslis comments were valid, try to follow them as much as possible. This is an exceptional case.